gilt/config.py

```diff
@@ -59,6 +63,35 @@ def config(filename):
     return [Config(**d) for d in _get_config_generator(filename)]
 
 
+def _parse_repo_uri(uri, scm="git"):
```

Comment:
We are going to need unit tests for this.  At a minimum ensuring it has parity with the library it is replacing.

Reply:
Done in latest update to this PR!